src/types/index.ts

```diff
@@ -385,7 +385,7 @@ export interface ITransaction {
   pending: boolean; // There are cases that not only the IPendingTransaction can be pending
   rawTx?: any; // TODO find type
```

Comment:
RawTx type is available in the SDK

Reply:
This change would require a deep refactoring in the whole project.